utils.py

```diff
@@ -35,6 +39,14 @@
 CLINIC_PATH = os.path.join(path_home, "CLINIC.csv")
 
 
+HOSPITAL_JSON = os.path.join(path_home, "HOSPITAL.json")
+HOTEL_JSON = os.path.join(path_home, "HOTEL.json")
+LOGISITICAL_JSON = os.path.join(path_home, "LOGISTICAL.json")
```

Comment:
Typo here. 
Also better to rename it as `LOGISTICS_JSON` as well as the json file name.

Reply:
ok